bower_components/jasmine-core/lib/jasmine-core/node_boot.js

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright (c) 2008-2015 Pivotal Labs
+
+Permission is hereby granted, free of charge, to any person obtaining
+a copy of this software and associated documentation files (the
+"Software"), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the following conditions:
+
+The above copyright notice and this permission notice shall be
+included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+module.exports = function(jasmineRequire) {
+  var jasmine = jasmineRequire.core(jasmineRequire);
+
+  var consoleFns = require('../console/console.js');
+  consoleFns.console(consoleFns, jasmine);
+
+  var env = jasmine.getEnv();
+
+  var jasmineInterface = jasmineRequire.interface(jasmine, env);
+
+  extend(global, jasmineInterface);
+
+  function extend(destination, source) {
+    for (var property in source) destination[property] = source[property];
```

Comment:
Expected '{' and instead saw 'destination'.<br>The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.